setup.cfg

```diff
@@ -0,0 +1,2 @@
+[mypy]
+ignore_missing_imports = True
```

Comment:
You shouldn't need this. Whenever there is an import that we shouldn't check, add the comment:

```python
from telegram import Bot  # type: ignore
```

Reply:
This ignores missing imports for third party libraries, so we don't have to add it in every single line.